src/plugin/sdl/sdl.c

```diff
@@ -370,16 +395,41 @@ static void SDL_update(void)
 
 static void SDL_redraw(void)
 {
-#ifdef X_SUPPORT
-  if (x11_display && !use_bitmap_font && vga.mode_class == TEXT) {
+  if (vga.mode_class == TEXT) {
     redraw_text_screen();
     return;
   }
-#endif
 
   do_redraw_full();
 }
 
+static void SDL_resize_to_fontsize(int xhint, int yhint)
```

Comment:
Oh, unless there is a good reason to do that,
can this be dropped? So that we only resize to
a window size, but never to the font size?

Reply:
renamed as below